browser/ui/content_settings/brave_content_setting_image_models.cc

```diff
@@ -7,7 +7,9 @@
 #include "brave/browser/ui/content_settings/brave_autoplay_blocked_image_model.h"
 #include "third_party/widevine/cdm/buildflags.h"
 
-#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
+// On Windows and MacOS, widevine is supported by component updater.
```

Comment:
Not sure if the comment is relevant here, maybe move it to the `widevine.gni` patch?

Reply:
Yep, `widevine.gni` is better place for this comment.
Done.